CHANGELOG.md

```diff
@@ -461,6 +461,7 @@ We skipped 24.2.0 because a draft was accidentally published. Please use `24.3.0
 - `[jest-worker]`: Migrate to TypeScript ([#7853](https://github.com/facebook/jest/pull/7853))
 - `[jest]`: Migrate to TypeScript ([#8024](https://github.com/facebook/jest/pull/8024))
 - `[pretty-format]`: Migrate to TypeScript ([#7809](https://github.com/facebook/jest/pull/7809), [#7809](https://github.com/facebook/jest/pull/7972))
+- `[jest-worker]`: Update queue workflow in Farm to improve performance and clearer ([#9521](https://github.com/facebook/jest/pull/9521)
```

Comment:
should be under `performance` at the top of the file (under master)

Reply:
I updated it